composer.json

```diff
@@ -39,7 +39,10 @@
     "require": {
         "php": "^5.5.0|^7.0",
         "payum/iso4217": "~1.0",
-        "guzzlehttp/guzzle": "~6.0",
+        "php-http/httplug": "^1.0",
```

Comment:
You are correct. This package is required indirectly. This row will not have any effect in practice. 

The reason I added it is because I believe it is good practice to specify all your dependencie and not rely on other packages to pull in the dependencies. We are using classes/interfaces from this package and that is why I've specified it as a dependency. 

I've noticed that this is very opinionated topic and I'm fine with removing it if you like.